packages/vscode/src/extension.ts

```diff
@@ -20,11 +19,11 @@ function createLanguageServer(
 }
 
 export function activate(context: ExtensionContext) {
-  // const serverModule = require.resolve('@prisma/language-server/bin/server.js')
+  const serverModule = require.resolve('@prisma/language-server/dist/src/cli')
```

Comment:
Curious, why is the folder called `cli`?

Reply:
for command line interface

Reply:
I know that but is CLI interface and server the same thing? This is kinda confusing. LSP can be exposed via CLI but here it is being exposed as a server right? But let's clean this up later.

Reply:
the CLI starts the server with the command `startServer()` from server.

Reply:
Hence, the confusion, it is not intuitive that CLI will start a server

Reply:
Can you please create an issue for this so we can clean this up later?